src/dotnet/APIView/APIView/CodeFileRenderer.cs

```diff
@@ -56,6 +61,19 @@ private void Render(List<CodeLine> list, IEnumerable<CodeFileToken> node, bool s
                         isDeprecatedToken = false;
                         break;
 
+                    case CodeFileTokenKind.SkipDiffRangeStart:
+                        // Skip only when creating diff text
+                        // Tokens should not be skipped from html renderring
+                        if (GetType() == typeof(CodeFileRenderer))
```

Comment:
Let's at least encapsulate this logic in a virtual property.

Reply:
Done.